complaint_search/tests/test_es_interface.py

```diff
@@ -171,8 +171,8 @@ def test_search_no_param__valid(self, mock_rget, mock_scroll, mock_count, mock_s
         act_body = mock_search.call_args_list[0][1]['body']
         diff = deep.diff(act_body, body)
         if diff:
-            print "search no param"
-            diff.print_full()
+            print "search no param"  # pragma: no cover
+            diff.print_full()  # pragma: no cover
```

Comment:
It might be better to just exclude test files from coverage, which was the intention of line 14 in `.coveragerc` originally

Reply:
I recently changed my stance on this, which led me to intentionally remove that line from the `coveragerc`. 

I like including test files in the coverage report as a quality control measure. If any part of a test file is being skipped in a test run, Coverage will automatically inform us. So, if I do something silly like mis-format a test file so some don't get recognized (or whatever), coverage will catch it.

Reply:
But if the `#pragma: no cover` is too ugly, I could just remove those `print` statements entirely. They only run if the tests fail (and aren't particularly helpful in that case, imo).

Reply:
How about
1. Removing the `print` statements (agreed they don't do much)
2. Add `diff.print_full()` to `.coveragerc` omits

Reply:
Oh, I meant I would remove the entire `if` block that gets run if the test fails. But this compromise works for me.